packages/shared/src/contexts/PaymentContext.tsx

```diff
@@ -80,7 +78,6 @@ export const PaymentContextProvider = ({
   const { data: planTypes } = useQuery({
     queryKey: generateQueryKey(RequestKey.PlanTypes),
     queryFn: getPricingIds,
-    enabled: showPlusSubscription && !isPlus,
```

Comment:
We gonna do the !isPlus rather on the page?

(showing something different, we waiting for design I guess)

Reply:
Yeah, but lets wait for design, so that we can still use this page in preview without having to mess with DB each time :)